includes/admin_import.php

```diff
@@ -167,10 +167,40 @@ function tsml_import_page()
 			]);
 			if (is_array($response) && !empty($response['body']) && ($body = json_decode($response['body'], true))) {
 
-				//if already set, hard refresh
+				/*When a data source already exists we want to apply changes detected to the local db */  
+
+				//initialize variables 
+				$feed_updates = $db_ids_to_delete = $message_lines = [];
+				$message = "<br><h2>Local Database Updates Being Applied From the $data_source_name Feed</h2><br>";
```

Comment:
be great to localize this string in a `__()`

Reply:
Absolutely. Will do.

Reply:
Done